ansible/configs/kni-osp/files/ocp-install.sh.j2

```diff
@@ -91,6 +91,12 @@ build_installer() {
   cd $HOME/go/src/github.com/openshift
   git clone --single-branch --branch release-$SUBVER https://github.com/openshift/installer.git
   cd installer/
+  cp -f go.mod go.bak
+  sed -i 's/library-go.*/library-go v0.0.0-20200917093739-70fa806b210a/g' go.mod
+  if [[ $(grep library-go go.mod | wc -l) = 1 ]]
+  then
+    sed -zEi 's|(\n[^\n]*){2}$|\n\tgithub.com/openshift/library-go => github.com/openshift/library-go v0.0.0-20200917093739-70fa806b210a&|' go.mod
+  fi
```

Comment:
As discussed with @rdoxenham let's try to build the installer outside.
Rhys will build the binary, and send it to me and i will host it in our Public S3 bucket.

Then in the script it will be as simple as `wget` or `curl`.

RHTR is very close but It will be more reliable and i think it's worth it.